lang/es/index.md

```diff
@@ -1,104 +1,237 @@
 
-Si todo esto suena deseable, todo lo que tienes que hacer para comenzar usando Versionado Semántico es declarar que lo estás haciendo y seguir las reglas. Haz un link a este sitio desde tu README para que otros conozcan las reglas y se puedan beneficiar de ellas.
+1. La precedencia se refiere a cómo las versiones se comparan entre ellas cuando se ordenan. 
 
-FAQ
----
+   1. La precedencia DEBE ser calculada separando los identificadores de la versión en mayor, menor, parche y prelanzamiento (dejando de lado los metadatos de compilación). 
+   2. La precedencia es determinada por la primera diferencia al comparar cada uno de los identificadores de izquierda a derecha como se indica: mayor, menor y parche son siempre comparadas numéricamente. 
+
+      Por ejemplo: 1.0.0 < 2.0.0 < 2.1.0 < 2.1.1. 
+    
+   3. Cuando la versión mayor, menor y parche son iguales, la versión de prelanzamiento tiene menor precedencia que la versión normal. 
+   
+      Por ejemplo: 1.0.0-alpha < 1.0.0. 
+      
+   4. La precedencia para dos versiones de prelanzamiento con la misma versión mayor, menor y parche DEBE ser determinada comparando cada identificador separado por punto, de izquierda a derecha, hasta que una diferencia sea encontrada como se indica: 
+
+      1. identificadores compuestos solamente por números son comparados numéricamente
+
+      1. los identificadores con letras o guiones son comparados léxicamente en orden ASCII.
+
+      1. Identificadores numéricos siempre tienen menor precedencia que identificadores no numéricos. 
+
+      1. Un conjunto de campos de prelanzamiento más numeroso tiene mayor precedencia que un conjunto menos numeroso, si todos los identificadores anteriores son iguales. 
+
+      Por ejemplo: 1.0.0-alpha < 1.0.0-alpha.1 < 1.0.0-alpha.beta < 1.0.0-beta < 1.0.0-beta.2 < 1.0.0-beta.11 < 1.0.0-rc.1 < 1.0.0.
+
+Forma Gramatical Backus–Naur para Versiones Válidas SemVer
+----------------------------------------------------------
+
+```
+<semver válido> ::= <versión núcleo>
+                  | <versión núcleo> "-" <prelanzamiento>
+                  | <versión núcleo> "+" <compilación>
+                  | <versión núcleo> "-" <prelanzamiento> "+" <compilación>
+
+<versión núcleo> ::= <major> "." <menor> "." <parche>
+
+<major> ::= <identificador numérico>
```

Comment:
I missed this one. Every occurrence of `major` should be replaced with `mayor`

Reply:
Fixed in new patch.